pkg/controllers/status/cluster_status_controller.go

```diff
@@ -368,7 +394,9 @@ func getKubernetesVersion(clusterClient *util.ClusterClient) (string, error) {
 func getAPIEnablements(clusterClient *util.ClusterClient) ([]clusterv1alpha1.APIEnablement, error) {
 	_, apiResourceList, err := clusterClient.KubeClient.Discovery().ServerGroupsAndResources()
 	if err != nil {
-		return nil, err
+		// if we fail, just log it.
+		// In case discovery client return part of succeed groups: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/discovery/discovery_client.go#L257
```

Comment:
should use absolute commit
https://github.com/kubernetes/kubernetes/blob/33de444861d3de783a6618be9d10fa84da1c11b4/staging/src/k8s.io/client-go/discovery/discovery_client.go#L257